pom.xml

```diff
@@ -26,7 +26,7 @@
     <artifactId>enrichment</artifactId>
 
     <name>EnrichmentTable</name>
-    <version>1.0.0</version>
+    <version>1.0.2-SNAPSHOT</version>
```

Comment:
I had updated it as we are resolving V2 issues. Please let me know if I should keep the version of the previous one.

Reply:
1.0.1 should work for now I would think as 1.0.0 has already been launched. @AlexanderPico can suggest what should be the version number we should go ahead with.